partials/footer-licence.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * The licensing message for our theme
+ * This is the template that displays the default navigation in the footer region
+ *
+ * @link      https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ * ©Crown Copyright 
+ * @version   1.0 24th February 2021
```

Comment:
Does this want to say @version 2.0? (Could be me missing something about file versioning!)

Reply:
Do we even want/need the version here?

Reply:
Don't know whether we need a version here - I'd probably say we don't.  But this is a new file, so version 1 I thought would be right.